config/webpack.config.prod.js

```diff
@@ -415,7 +415,7 @@ module.exports = {
                         // it's runtime that would otherwise be processed through "file" loader.
                         // Also exclude `html` and `json` extensions so they get processed
                         // by webpacks internal loaders.
-                        exclude: [/\.(js|mjs|jsx|ts|tsx)$/, /\.html$/, /\.json$/],
+                        exclude: [/\.(js|mjs|jsx|ts|tsx|snap)$/, /\.html$/, /\.json$/],
```

Comment:
I know you didn't make this change, but is there any reason why `.html` and `.json` extensions are listed as separate regex strings?  Couldn't the regex just add `|snap|html|json` to the end?

Reply:
I think we just inherited this line from here:

https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/config/webpack.config.js#L507. It's strangely broken out separately there as well.

Reply:
Yeah, was just more curious than anything. 😄

Reply:
I've moved them around and gotten the same results, but honestly I wasn't sure the reasoning for them being separated so didn't want to change it. I think we do need a follow up story to remove all the unnecessary stuff from the webpack configs. They're really unwieldy and have slowed down my progress on the CSS modules story a lot.